backend/main.py

```diff
@@ -28,6 +29,10 @@ async def school(request):
     return JSONResponse({'lesson': 'software engineering'})
 
 
+async def tomasz(request):
+    return RedirectResponse({'https://www.youtube.com/watch?v=AFhZxua-2P8': 303})
```

Comment:
Chyba niestety nie działa:
Po wejściu na localhost/tomasz wynik przekierowania to:
`http://localhost:8000/%7B'https://www.youtube.com/watch?v=AFhZxua-2P8%27:%20303%7D`
A przeglądarka zwraca:
`Not Found`

Reply:
do not use polish